src/test/java/PlayListTests.java

```diff
@@ -28,9 +20,9 @@ public void createNewList () {
         //WHEN
         playListPage.playListArea();
         playListPage.addingPlaylistbtn();
-        playListPage.enterPlayListName(playListName);
+        playListPage.enterPlayListName("my list");
         //THEN
-        Assert.assertEquals(playListPage.getConfirmationPopupText(),"Created playlist "+ "\""+ playListName+"."+"\"" );
+        Assert.assertEquals(playListPage.getConfirmationPopupText(), "Created playlist " + "\"" + "my list" + "." + "\"");
```

Comment:
Test failed!

You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/211770085-19835fe7-e1b0-4223-86dd-a92183341a25.png)

Reply:
I fixed it.